prow/tide/tide.go

```diff
@@ -610,7 +664,7 @@ func filterSubpool(ghc githubClient, mergeAllowed func(*PullRequest) (string, er
 //   status is preventing merge. Required ProwJob statuses are allowed to be
 //   'pending' because this prevents kicking PRs from the pool when Tide is
 //   retesting them.)
-func filterPR(ghc githubClient, mergeAllowed func(*PullRequest) (string, error), sp *subpool, pr *PullRequest) bool {
+func filterPR(ghc githubClient, mergeAllowed func(*CodeReviewCommon) (string, error), sp *subpool, pr *CodeReviewCommon) bool {
```

Comment:
As with initSubpoolData, I expect the filterPR function to be a function in the controller interface due to the differences between Gerrit and GitHub.